CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file. This
 change log follows the conventions of
 [keepachangelog.com](http://keepachangelog.com/).
 
+## 3.8.7 / 2023-08-31
+- [Experimental] add experimental feature to not print the entire actual data-structure on
```

Comment:
marking it as experimental because I haven't really tested it much and would be good to get some user feedback before committing to the feature. Sound alright or someone have another idea?